src/lib/containers/home_page/featured-data/FeaturedDataPlots.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react'
+import QueryWrapperFacetPlotsCard, { QueryWrapperFacetPlotsCardProps } from './QueryWrapperFacetPlotsCard'
+import { useEffect, useState } from 'react'
+
+export type FeaturedDataPlotsProps = {
+  token?: string
+  configs: QueryWrapperFacetPlotsCardProps[]
+  rgbIndex?: number
+  exploreSql?: string,
+  explorePagePath?: string,
+}
+
+const FeaturedDataPlots: React.FunctionComponent<FeaturedDataPlotsProps> = props => {
```

Comment:
May end up exporting FeaturedDataPlots too, if other portals don't require tabs.  This should be a quick fix if we need to do it.